CodePush.h

```diff
@@ -88,6 +88,20 @@ failCallback:(void (^)(NSError *err))failCallback;
 
 @end
 
+@interface CodePushStatusReport : NSObject
+
++ (NSString *)getDeploymentKeyFromStatusReportIdentifier:(NSString *)statusReportIdentifier;
```

Comment:
We can probably remove all of the internal methods from the interface, since the `CodePush` class doesn't consume them.